src/components/Link/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styles from "./index.module.css";
+
+const Link = ({ href, children }) => (
```

Comment:
We need to distinguish external links by internal links `import Link from next/link`

Reply:
Nice catch, I'll change

Reply:
Updated